packages/commonwealth/server/controllers/server_chains_methods/get_related_communities.ts

```diff
@@ -0,0 +1,43 @@
+import { QueryTypes } from 'sequelize';
+import { sequelize } from '../../database';
+import { ServerChainsController } from '../server_chains_controller';
+
+export type GetRelatedCommunitiesOptions = { base: string };
```

Comment:
- Can you give me an example of `base` ? It is name like `dydx` (probably not) or some id like 5 or '5'?

- I think I should be able to send two parameters to get data for this view 

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/126f2015-8abe-4be5-a00a-f481ea1fccc9)

1. Chain base - Required - based on this param the list of communities is created
2. Community name - optional - used when user types something in searchbar to look for community by its name

Reply:
base would be something like:
`Ethereum (Mainenet)` (largest one)

Will implement community name filtering and thumbs up comment when finished